scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -5,41 +5,45 @@
 """
 import logging
 import os
-import sys
 import subprocess
+import sys
 from clusters import GKECluster
+from collections import namedtuple
 from compatibility_test import make_compatibility_test_runner
 from get_latest_helm_chart_versions import get_latest_helm_chart_versions
 
+# start logging
+logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
 
 # set required test parameters
 os.environ["ORCHESTRATOR_FLAVOR"] = "k8s"
 os.environ["ROX_POSTGRES_DATASTORE"] = "true"
 
 central_chart_versions = get_latest_helm_chart_versions("stackrox-central-services", 2)
 sensor_chart_versions = get_latest_helm_chart_versions("stackrox-secured-cluster-services", 3)
-latest_tag = subprocess.check_output("make tag", shell=True).strip().decode("utf-8")
+makefile_path = os.path.abspath(os.path.join(os.path.dirname( __file__ ), '../../..'))
+latest_tag = subprocess.check_output(["make", "tag", "-C", makefile_path, "--quiet", "--no-print-director"], shell=False, encoding='utf-8').strip()
 
 if len(central_chart_versions) == 0:
     raise RuntimeError("Could not find central chart versions.")
-# Latest central vs last 4 sensor versions
-test_tuples = [[latest_tag, sensor_chart_versions[i]] for i in range(0, len(sensor_chart_versions))]
-# Latest sensor vs 1 version older central
-if len(central_chart_versions) > 1:
-    test_tuples.append([central_chart_versions[1], latest_tag])
+if len(sensor_chart_versions) == 0:
+    raise RuntimeError("Could not find sensor chart versions.")
 
-logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
+Chart_versions = namedtuple("Chart_versions", ["central_version", "sensor_version"])
 
-chart_versions = get_latest_helm_chart_versions("stackrox-secured-cluster-services")
+# Latest central vs sensor versions in sensor_chart_versions (latest 4 releases)
```

Comment:
Is it for sure `4 releases`?
I suggest to omit the count completely because one can find it in the code above.

Reply:
Okay